gateway/src/main/java/io/camunda/zeebe/gateway/impl/probes/health/ResponsiveHealthIndicatorProperties.java

```diff
@@ -14,16 +14,24 @@
 @ConfigurationProperties(prefix = "management.health.gateway-responsive")
 public class ResponsiveHealthIndicatorProperties {
 
-  private Duration requestTimeout = Duration.ofMillis(500);
+  private HealthZeebeClientProperties healthZeebeClientProperties =
+      new HealthZeebeClientProperties();
 
-  public Duration getRequestTimeout() {
-    return requestTimeout;
+  public ResponsiveHealthIndicatorProperties() {
+    healthZeebeClientProperties.setRequestTimeout(Duration.ofMillis(500));
   }
 
+  public HealthZeebeClientProperties getHealthZeebeClientProperties() {
+    return healthZeebeClientProperties;
+  }
+
+  public void setHealthZeebeClientProperties(
+      final HealthZeebeClientProperties healthZeebeClientProperties) {
+    this.healthZeebeClientProperties = healthZeebeClientProperties;
+  }
+
+  @Deprecated(forRemoval = true, since = "8.1.0")
```

Comment:
```suggestion
  @Deprecated(forRemoval = true, since = "8.2.0")
```

❌ Sorry for taking so long, now we have to switch this deprecation version 😅

Reply:
Thanks, fixed :)